### ISSUE TITLE
Multiple binaries share the same codesign path

### ISSUE THREAD

**Issue by VictoriaKostyleva** [cossacklabs/themis#940]
### Steps to reproduce:
1. Create new project
2. Add package dependency to your project. We are using SPM
3. Archive build
4. Try "Distribute app"
5. See error:
```
Multiple binaries share the same codesign path:
/Applications/MyApp.app/Contents/Frameworks/themis.framework/Versions/Current
Binaries:
/Applications/MyApp.app/Contents/Frameworks/themis.framework/Versions/Current/themis
/Applications/MyApp.app/Contents/Frameworks/themis.framework/themis
This can happen if your build process copies frameworks by following symlinks.
```
<img width="754" alt="Screenshot" src="https://user-images.githubusercontent.com/23255940/186456164-302e3ddd-bb0c-4bae-9faa-58a33b11c908.png">

**Expected behavior**
Exporting should work fine

**Actual behavior**
We receive an error about multiple binaries sharing the same codesign path:

**Environment:**
- Platform: macOS 12.5
- Themis version: 0.14.8
- Installation way: Swift Package Manager

**Comment by vixentael** [comment]
Ah the beauty of SPM.. Always exciting challenges 😅

Thank you, @VictoriaKostyleva! Could you please help us to narrow down the issue?

1. Please specify Xcode version you are using?
2. Do I understand correctly, that you are building app for macOS, not iOS? (Or do you build for both?)
3. Do you see this happening for M1 or Intel macbook?
4. Does the error happen only during Distribution? Build and run in Debug and Release are working fine? 

Basically, any details would help us to understand the root cause. 

I've googled, and people do complain about similar behavior, I'll put the links for Themis team here:
https://github.com/mapbox/mapbox-gl-native/pull/5850/files

cc @julepka @radetsky @ilammy

**Comment by VictoriaKostyleva** [comment]
@vixentael 

1. Xcode version is 13.4.1 and 14.0 Beta 4
2. Yes, we are building app only for for macOS
3. It happens for M1, have not tried for Intel MacBook.
4. Yes, it happen only during Distribution. Build and run in Debug and Release are working fine.

We solved this issue by downloading your repo, updating what Xcode 14.0 Beta 4 suggested, making local framework and putting it into our app.
Here is what Xcode suggested:
<img width="1028" alt="Screenshot" src="https://user-images.githubusercontent.com/23255940/186616606-24651a40-3cf6-4165-b00d-7bdca2c20254.png">
.

**Comment by vixentael** [comment]
Fantastic, thank you @VictoriaKostyleva

We have reproduced the issue, will work on fix, 🤞, update the version, release and let you know.

**Comment by vixentael** [comment]
This will be fixed in upcoming 0.14.10 release. ETA: monday.

**Comment by vixentael** [comment]
@VictoriaKostyleva we have release 0.14.9 where addressed this issue
https://github.com/cossacklabs/themis/releases/tag/0.14.9

could you please verify and let us know if it's working for you?

**Comment by VictoriaKostyleva** [comment]
@vixentael Yeah, it works now. Thank you!
